applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -397,32 +377,37 @@ To connect with an external MCU using UART_2, change the configuration files for
 
 * In the :file:`nrf9160dk_nrf9160_ns.conf` file::
 
-   # Use UART_0 (when working with PC terminal)
-   # unmask the following config
-   #CONFIG_SLM_CONNECT_UART_0=y
-   #CONFIG_UART_0_NRF_HW_ASYNC_TIMER=2
-   #CONFIG_UART_0_NRF_HW_ASYNC=y
-   #CONFIG_SLM_WAKEUP_PIN=6
-   #CONFIG_SLM_INDICATE_PIN=2
+  .. code-block:: console
+
+     # Use UART_0 (when working with PC terminal)
+     # unmask the following config
+     #CONFIG_UART_0_NRF_HW_ASYNC_TIMER=2
+     #CONFIG_UART_0_NRF_HW_ASYNC=y
+     #CONFIG_SLM_WAKEUP_PIN=6
+     #CONFIG_SLM_INDICATE_PIN=2
 
-   # Use UART_2 (when working with external MCU)
-   # unmask the following config
-   CONFIG_SLM_CONNECT_UART_2=y
-   CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
-   CONFIG_UART_2_NRF_HW_ASYNC=y
-   CONFIG_SLM_WAKEUP_PIN=31
-   CONFIG_SLM_INDICATE_PIN=30
+     # Use UART_2 (when working with external MCU)
+     # unmask the following config
+     CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
+     CONFIG_UART_2_NRF_HW_ASYNC=y
+     CONFIG_SLM_WAKEUP_PIN=31
+     CONFIG_SLM_INDICATE_PIN=30
 
 
 * In the :file:`nrf9160dk_nrf9160_ns.overlay` file::
 
-   &uart0 {
+  .. code-block:: console
+
+     / {
+	chosen {
+		zephyr,slm-uart = &uart2;
+
```

Comment:
It was missing the trailing `}` and you seem to have preserved that "_convention_". However, now that there are two levels of `{` I would prefer to have the trailing `}`

Reply:
That's the shot quotes of chagnes to existing overlay file, not the full content

Reply:
What is the short quotation? The lack of trailing `}` seems really confusing to me.

Reply:
@wiba-nordic Is it common to use this sort of "short quotation"?

Reply:
updated as *full* quotation

Reply:
As far as I can recall, we have very few examples of using quotations for overlay files, so I couldn't tell if it's common to use short quotes or not.